libsecp256k1.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "libsecp256k1",
+    "config-opts": [ "--disable-static" ],
```

```suggestion
    "config-opts": [
        "--enable-module-recovery",
        "--disable-static"
    ],
```
Is needed for electrum 4.0.0.

---

ok, I'll commit newer commit and remove `--disable-static` in a hour

---

Disable static is fine, add ` "--enable-module-recovery",`.

---

Yes, sorry.

---

added suggestion in 3bdca52